packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -340,6 +342,9 @@ export default class Meetings extends WebexPlugin {
     if (newLocus) {
       const isNewLocusAsBreakout = MeetingsUtil.isBreakoutLocusDTO(newLocus);
       const isSelfMoved = newLocus?.self?.state === _LEFT_ && newLocus?.self?.reason === _MOVED_;
+      const isSelfMovedToLobby =
+        newLocus?.self?.devices[0]?.intent?.reason === _ON_HOLD_LOBBY_ &&
```

Comment:
suggestion: you could use lodash's get here to provide a safe way of getting the reason without all those `?`
i.e.
```js
import {get} from 'lodash'

const isSelfMovedToLobby = get(newLocus, 'self.devices[0].intent.reason') === _ON_HOLD_LOBBY && 
get(newLocus, 'self.devices[0].intent.type') === _WAIT_; 

```

Reply:
Thanks for your suggestion